geolocator.ipynb

```diff
@@ -558,9 +608,14 @@
         "                url_plot = gr.Plot()\n",
         "        url_text_button = gr.Button(\"Go locate!\")\n",
         "\n",
+        "    callback.setup(components = [img_input, img_text_output, img_text_output2],\n",
+        "                    flagging_dir = make_unique_bucket_name(prefix = application_and_bucket_name, seed = '420')\n",
+        "                    )\n",
```

Comment:
Rather than calling gantry from within the notebook, I think it'd be nice to call it from the services/gantry/app.py file or a separate file under services/gantry. What do you think?

Reply:
Makes sense! I'll go ahead and update it to call from services/ganty/app.py

Reply:
Wait, I think you mean services/gradio/app.py and I mean that too lol